assets/styles/_base.scss

```diff
@@ -29,6 +31,7 @@ html {
 }
 
 body {
+  height: 100vh;
```

Comment:
This creates an issue where the body is too large on iOS devices and thus scrollable. Using `height: 100%` instead here and `height` instead of `min-height` on html should fix that.

Just like we've had it before: https://github.com/flyntwp/flynt/commit/b75dc78fb385fca081fefa692e350920d74fb8f0 (In this commit we removed the sticky footer because we thought it was too specific, but it was working well that way.)